app.js

```diff
@@ -0,0 +1,93 @@
+const form = document.querySelector('#new-task-form');
+const button = document.querySelector('#button');
+const list =  document.querySelector('#task-list');
+const field = form['new-task-input'];
+
```

Comment:
Let's start to modularise our codebase. 
We'll create another file that'll be responsible for target DOM elements and export them. We'll use them here by importing.

Also avoid giving generic names to DOM elements like `list` or `field`. 
Postfix `elements` or `$` to indicate that, the variable contains HTML DOM Element. e.g: `todoListElement` or `todoList$`

```js
// domElements.js
export const todoSubmitButton = document.querySelector('#todo-submit-button')
```

```js
// app.js
import { todoSubmitButton } from "./domElements.js"

// continue with other code
```